hedera-node/hedera-app/src/xtest/java/contract/BurnsXTest.java

```diff
@@ -68,6 +71,18 @@
 import org.apache.tuweni.bytes.Bytes;
 import org.jetbrains.annotations.NotNull;
 
+/**
+ * Exercises burnToken on a fungible and non-fungible token via the following steps relative to an {@code OWNER} account:
+ * <ol>
+ *     <li>Burns {@code ERC20_TOKEN} via BURN_TOKEN_V1 operation</li>
+ *     <li>Burns {@code ERC20_TOKEN} via BURN_TOKEN_V2 operation</li>
+ *     <li>Burns {@code ERC721_TOKEN} via BURN_TOKEN_V1 operation</li>
```

Comment:
Make sure to keep the same order as the calls to prevent questions and errors when revisiting this test.